users/userRouter.js

```diff
@@ -1,47 +1,140 @@
-const express = 'express';
+const express = require("express");
 
-const router = express.Router();
+const Users = require("./userDb.js");
 
-router.post('/', (req, res) => {
+const router = express.Router();
 
+router.post("/", validateUser, (req, res) => {
+  const user = req.body;
+  Users.insert(user)
+    .then(user => {
+      res.status(201).json(user);
+    })
+    .catch(err => {
+      res.status(500).json({
+        message: "cannot post this user idk"
+      });
+    });
 });
 
-router.post('/:id/posts', (req, res) => {
-
+router.post("/:id/posts", validateUserId, validatePost, (req, res) => {
+  const user = req.body;
```

Comment:
I know this was put here by default, but do you think this belong here or in postRoutes?